src/utils/tableSort.test.ts

```diff
@@ -0,0 +1,168 @@
+import { HeaderGroup } from 'react-table';
+import { DateTime } from 'luxon';
+
+import {
+  currentTableSortDescription,
+  getColumnSortStatus,
+  getHeaderSortIcon,
+  sortColumnProps,
+  sortColumnValues
+} from './tableSort';
+
+describe('tableSortUtil', () => {
+  describe('getColumnSortStatus', () => {
+    it('return text - ascending', () => {
+      const column = {
+        isSorted: true,
+        isSortedDesc: false
+      } as HeaderGroup;
+      expect(getColumnSortStatus(column)).toEqual('ascending');
+    });
+
+    it('return text - descending', () => {
+      const column = {
+        isSorted: true,
+        isSortedDesc: true
+      } as HeaderGroup;
+      expect(getColumnSortStatus(column)).toEqual('descending');
+    });
+
+    it('return text - none', () => {
+      const column = {
+        isSorted: false,
+        isSortedDesc: true
+      } as HeaderGroup;
+      expect(getColumnSortStatus(column)).toEqual('none');
+    });
+  });
+
+  describe('getHeaderSortIcon', () => {
```

Comment:
For the assertions in these tests, it might be better to use `expect().toMatch([regex pattern])` and just look for the relevant classes, rather than asserting on the classes like `position-absolute` that don't really pertain to sorting.